system_metrics_collector/src/topic_statistics_collector/subscriber_topic_statistics.hpp

```diff
@@ -97,6 +97,7 @@ class SubscriberTopicStatisticsNode : public system_metrics_collector::MetricsMe
 
     auto callback = [this](typename T::SharedPtr received_message) {
         for (const auto & collector : statistics_collectors_) {
+          RCLCPP_DEBUG(this->get_logger(), collector->GetStatusString());
```

Comment:
drop?

Reply:
I thought `PeriodicMeasurementNode` provided sufficient logging data / context when set to debug, so this might be duplicative.

Reply:
What Devin said: this will mirror `PeriodicMeasurementNode`'s behaviour and help with debugging.